usaspending_api/awards/v2/urls_awards.py

```diff
@@ -13,10 +13,17 @@
     url(r"^funding/$", AwardFundingViewSet.as_view()),
     url(r"^funding_rollup/$", AwardFundingRollupViewSet.as_view()),
     url(r"^last_updated", AwardLastUpdatedViewSet.as_view()),
-    url(r"^(?P<requested_award>[A-Za-z0-9_. -]+)/$", AwardRetrieveViewSet.as_view()),
-    url(r"^count/transaction/(?P<requested_award>[A-Za-z0-9_. -]+)/$", TransactionCountRetrieveViewSet.as_view()),
-    url(r"^count/subaward/(?P<requested_award>[A-Za-z0-9_. -]+)/$", SubawardCountRetrieveViewSet.as_view()),
     url(
-        r"^count/federal_account/(?P<requested_award>[A-Za-z0-9_. -]+)/$", FederalAccountCountRetrieveViewSet.as_view()
+        "^count/transaction/(?P<requested_award>(((CONT|ASST)_(AWD|IDV|NON|AGG)_.+)|([0-9]+)))/$",
```

Comment:
It would be better to define the regex in one location and use that var for the URLs

Reply:
Also, shouldn't those strings be prefixed by `r`?

Reply:
`r` indicates a raw string and is usually only used if you want to prevent escaping to simplify syntax (e.g. `"this \\n is \\t an \\r example" == r"this \n is  \t an \r example"`)

Reply:
yeah, the literal strings were removed before when we were limiting by individual characters, should be fixed now